glotpress.php

```diff
@@ -35,6 +35,35 @@
 define( 'GP_PLUGIN_FILE', __FILE__ );
 define( 'GP_PATH', dirname( __FILE__ ) . '/' );
 define( 'GP_INC', 'gp-includes/' );
+define( 'GP_WP_REQUIRED_VERSION', '4.4' );
+
+// This adds a row after GlotPress in the plugin page IF an incompatible version of WordPress is running.
+function gp_admin_notices() {
+	global $wp_version;
+	
+	$screen = get_current_screen();
+	
+	if( 'plugins' == $screen->id ) {
+?>
+    <div class="error">
+		<p><?php _e( 'GlotPress Disabled &#151; You are running an unsupported version of WordPress.', 'glotpress' ); ?></p>
+
+		<p><?php echo sprintf( __( 'GlotPress requires WordPress %s or later and has detected you are running %s. Upgrade your WordPress install or deactivate the GlotPress plugin to remove this message.', 'glotpress' ), GP_WP_REQUIRED_VERSION, $wp_version ); ?></p>
```

Comment:
I originally had both in one paragraph but it was too much of a text blob, so I broke in to two.  If we leave it as two, the paragraph break is the right thing to do as just a line break still creates a hard to read text block.

I'll add the numbered placeholders and add the esc_html() calls.